pkg/storageincentives/events.go

```diff
@@ -17,6 +17,8 @@ const (
 	claim
 	sample
 	sampleEnd
+	winner
```

Comment:
these are the different phases, so these new values do not belong here

Reply:
Where should they blong?

Reply:
so instead of passing `PhaseType` to `SetPhase`, i would simply pass a string, or redefine these "phases" in the nodestatus.go under a different name.